app/services/to_form/contact_emails_mapper.rb

```diff
@@ -4,15 +4,25 @@ module ToForm
   # Maps the value from the accessContact field in the Cocina::DescriptiveAccessMetadata object.
   # to the email value for a contact_email in the WorkForm
   class ContactEmailsMapper < BaseMapper
+    def initialize(cocina_object:, deposits_contact_email: nil)
+      @cocina_object = cocina_object
+      @deposits_contact_email = deposits_contact_email
+      super(cocina_object:)
+    end
+
     # @return [Array<Hash>, Nil] with keys for email
     def call
       return nil if cocina_object.description.access.blank?
 
       cocina_object.description.access.accessContact.filter_map do |access_contact|
-        next if access_contact.value.blank?
+        next if access_contact.value.blank? || access_contact.value == deposits_contact_email
 
-        { 'email' => access_contact[:value] }
+        { 'email' => access_contact.value }
       end.presence
     end
+
+    private
+
+    attr_reader :deposits_contact_email
```

Comment:
When mapping Cocina into the H3 form representation, keep the work and collection contact emails distinct.